src/main/scala/cromwell/engine/CromwellActor.scala

```diff
@@ -1,9 +1,26 @@
 package cromwell.engine
 
+import akka.event.LoggingAdapter
 import akka.util.Timeout
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
 trait CromwellActor {
-  implicit val timeout = Timeout(5 seconds)
+  protected implicit val timeout = Timeout(5 seconds)
+
+  protected val log: LoggingAdapter
+
+  /**
+   * Hack to log just an error message so Akka TestKit can pick up on it, and then again to log
```

Comment:
This is kind of horrible.  I'd like to spend some cycles today digging into this as I'm currently under the budgeted points.  :smile: